DEVELOPMENT.md

```diff
@@ -28,29 +35,95 @@ meteor
 This will start up a server on [http://localhost:3000](http://localhost:3000),
 which you can navigate to with your web browser of choice.
 
-## Create admin account
+## Running a production instance
+
+To run Jolly Roger on a server for lots of people to use, see the
+[Deployment and Monitoring] documentation for Meteor. There are many ways
+to run the server, but it boils down to taking the tarball generated by
+`meteor build`, untarring it on some server, setting environment variables
+(see below), and running `node main.js`.
+
+You will also need some dependent services, at the very least a
+Mongo DB instance (for `$MONGO_URL`), some way to send email (for `$MAIL_URL`),
+a web server that can forward to your server and `$PORT`, and some domain
+name pointing to that web server (hopefully with an SSL certificate).
+
+As a "canned configuration" for all of that,
+[`cloudformation/jolly-roger.yaml`](cloudformation/jolly-roger.yaml) is
+a template for [AWS CloudFormation](https://aws.amazon.com/cloudformation/).
+You will need to edit it for your use (change the domain name, etc) but then
+it can spin up a full production system including Jolly Roger, MongoDB,
```

Comment:
Small nit: mongodb still has to come from a 3rd party - we're using MongoDB's hosted service (Atlas)

Reply:
Fixed!